### DISCUSSION TITLE
Why does i18n support up to 100 languages?

### DISCUSSION THREAD

**Discussion by irmakcosarsahna** [vercel/next.js#28389] [Help]
Hello,

My site has support for 127 languages, but i18n can support 100 languages, is there a way to increase it?
And why are you limiting it?

**Comment by irmakcosarsahna** [comment]
Is there anyone who can help? I need a way to increase it

**Reply by leerob** [reply]
👋  How are you currently handling deployment/localization on your Next.js app?
